<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 127);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="txt_usi">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ede007;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 112);</string>
       </property>
       <property name="text">
        <string>Usu치rio ou senha inv치lida.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_erro">
       <property name="styleSheet">
        <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="content">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>681</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 127);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="login_area">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>9</y>
      <width>401</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 112);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/newPrefix/logo_login.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_usuario">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>280</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border: 2px solid #ede007 ;
	border-radius: 10px;
	padding: 10px;
	color: rgb(237, 224, 7);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>usu치rio</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_senha">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>280</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border: 2px solid #ede007 ;
	border-radius: 10px;
	padding: 10px;
	color: rgb(237, 224, 7);
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>senha</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox  {
	color: rgb(237, 224, 7)
}

QCheckBox:indicator  {
	border: 1px solid rgb(237, 224, 7);
	border-radius: 3px;
	width: 12px;	
	height: 12px;
}

QCheckBox::indicator:checked  {
	border: 1px solid rgb(237, 224, 7);
	background-color: rgb(237, 224, 7);
}

</string>
     </property>
     <property name="text">
      <string>lembrar usu치rio</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_acesso">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>280</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(237, 224, 7);
	border-radius: 10px;
	padding: 10px;
	color: rgb(0, 0, 112);
}

QPushButton:hover {
	background-color: rgb(187, 237, 6);
	border-radius: 10px;
	padding: 10px;
	color: rgb(0, 0, 112);
}

QPushButton:pressed {
	background-color: rgb(211, 214, 4);
}</string>
     </property>
     <property name="text">
      <string>acessar</string>
     </property>
    </widget>
    <zorder>txt_usuario</zorder>
    <zorder>txt_senha</zorder>
    <zorder>checkBox</zorder>
    <zorder>btn_acesso</zorder>
    <zorder>frame_3</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
